src/runtime/offers-flow.js

```diff
@@ -51,20 +52,54 @@ export class OffersFlow {
       isClosable = false; // Default is to hide Close button.
     }
 
+    const feArgsObj = {
+      'productId': deps.pageConfig().getProductId(),
+      'publicationId': deps.pageConfig().getPublicationId(),
+      'showNative': deps.callbacks().hasSubscribeRequestCallback(),
+      'productType': ProductType.SUBSCRIPTION,
+      'list': (options && options.list) || 'default',
+      'skus': (options && options.skus) || null,
+      'isClosable': isClosable,
+    };
+
+    if (options && options.oldSku) {
+      feArgsObj['oldSku'] = options.oldSku;
```

Comment:
I think this can go above like:
'oldSku': (options && options. oldSku) || undefined,

Reply:
Unfortunately, doing this breaks our tests because they expect the object to not have an `oldSku` property at all if `oldSku` isn't relevant. I guess we could modify all the tests?

Reply:
I see. In that case ignore this comment :)